DOCS-4883 Updates Error Tracking Screenshots (#17544)
DataDog/documentation: content/en/tracing/error_tracking/explorer.md

---
title: Error Tracking Explorer
kind: documentation
description: Learn about the Error Tracking Explorer.
further_reading:
- link: '/monitors/types/error_tracking'
  tag: 'Documentation'
  text: 'Learn about Error Tracking Monitors'
- link: '/tracing/error_tracking'
  tag: 'Documentation'
  text: 'Learn about Error Tracking for Backend Services'
---

## Overview

> The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical it is. 

Reviewer: ```suggestion
The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical they are. 
```

{{< img src="tracing/error_tracking/error_tracking_explore_inspect.png" alt="Error Tracking Explorer" style="width:100%;" >}}

## Explore your issues

> Each item listed in the Error Tracking Explorer surfaces high-level information about the issue, including the following:

Author: ```suggestion
Each item listed in the [Error Tracking Explorer][3] surfaces high-level information about the issue, including the following:
```

- All the error span tags, such as the resource or operation name, with direct access to the related trace or to logs that are linked to it.
- Insights about the health of the underlying host or container when this particular error occurred.

## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /tracing/trace_explorer/#live-search-for-15-minutes
[2]: /tracing/trace_pipeline/trace_retention/#retention-filters

> [3]: https://app.datadoghq.com/rum/error-tracking

Author: ```suggestion
[3]: https://app.datadoghq.com/apm/error-tracking
```
